packages/flame_studio/lib/src/core/game_controller.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flame/game.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final gameControllerProvider =
+    StateNotifierProvider<_GameController, _GameState>(
+  (ref) => _GameController(),
+);
+
+Future<Game?> _findGame() async {
+  await null;
```

Comment:
What does this do? Keeping it async? Maybe `FutureOr` could be used instead?

Reply:
This adds an async delay before the future is actually executed. Within that delay the widget tree will be built, allowing us to explore it.

Reply:
Ah, I see! Maybe better to trigger it from `WidgetsBinding.instance.addPostFrameCallback` instead?

Reply:
Yep, this is better.